GPU/TPCSpaceChargeBase/AliTPCSpaceCharge3DCalc.cxx

```diff
@@ -1401,29 +1440,21 @@ void AliTPCSpaceCharge3DCalc::InitSpaceCharge3DPoissonIntegralDz(
         matricesRIrregular = fMatrixRListIrregularC;
         matricesZIrregular = fMatrixZListIrregularC;
         matricesLookUpCharge = fMatrixChargeC;
+
         chargeInterpolator = fInterpolatorChargeC;
         potentialInterpolator = fInterpolatorPotentialC;
-        fLookupDistC->SetLookUpR(matricesDistDrDzC);
-        fLookupDistC->SetLookUpPhi(matricesDistDPhiRDzC);
-        fLookupDistC->SetLookUpZ(matricesDistDzC);
-        fLookupElectricFieldC->SetLookUpR(matricesErC);
-        fLookupElectricFieldC->SetLookUpPhi(matricesEPhiC);
-        fLookupElectricFieldC->SetLookUpZ(matricesEzC);
-
-        lookupLocalDist->SetLookUpR(matricesDistDrDzC);
-        lookupLocalDist->SetLookUpPhi(matricesDistDPhiRDzC);
-        lookupLocalDist->SetLookUpZ(matricesDistDzC);
-
-        lookupLocalCorr->SetLookUpR(matricesCorrDrDzC);
-        lookupLocalCorr->SetLookUpPhi(matricesCorrDPhiRDzC);
-        lookupLocalCorr->SetLookUpZ(matricesCorrDzC);
+        fLookupDistC->SetLookUpR(matricesDistDrDz);
+        fLookupDistC->SetLookUpPhi(matricesDistDPhiRDz);
+        fLookupDistC->SetLookUpZ(matricesDistDz);
+        fLookupElectricFieldC->SetLookUpR(matricesEr);
+        fLookupElectricFieldC->SetLookUpPhi(matricesEPhi);
+        fLookupElectricFieldC->SetLookUpZ(matricesEz);
 
         potentialBoundary = fListPotentialBoundaryC;
         f1BoundaryIFC = fFormulaBoundaryIFCC;
-        f1BoundaryOFC = fFormulaBoundaryOFCC;
-        f1BoundaryROC = fFormulaBoundaryROCC;
+        f1BoundaryOFC = fFormulaBoundaryOFCA;
+        f1BoundaryROC = fFormulaBoundaryROCA;
```

Comment:
Shouldn't these be the boundary formulas for C side since it is in the C side part of the condition?